_layouts/fellow.html

```diff
@@ -14,11 +14,9 @@
 <br>
 <br>
 <b> Fellowship dates: </b>
-{%- if page.dates.first -%}
-{{   page.dates | join: ", " }}
-{%- else -%}
-{{   page.dates }}
-{%- endif -%}
+{% for dates in page.dates %}
+  {{dates.start_date | date: "%b" }} - {{dates.end_date | date: "%b %Y"}}<br>
+{% endfor %}
```

Comment:
I think it would nice to keep the optional array, that is 

```
{%- if page.dates.first -%}
  {% for dates in page.dates %}
    {{ dates.start_date | date: "%b" }} - {{ dates.end_date | date: "%b %Y" }}<br>
  {% endfor %}
 {%- else -%}
  {{ page.dates.start_date | date: "%b" }} - {{ page.dates.end_date | date: "%b %Y" }}<br>
 {%- endif -%}
```

That way, you can forget the "-" before `start-date`. The other parts work like this.

Reply:
I see what you're saying.  Fellows will have a start_date and end date formatted as I indicate.  For cases like Reik, we then create the case with a "dates" array with pairs of start and end dates.  Then incorporate your idea to differentiate between the "single" case and the "multiple" case. I'm going to push up another revision.